cloudbuild/nightly/nightly.sh

```diff
@@ -76,7 +76,7 @@ run_read_write_test(){
   CLUSTER_FILE=$3
   PROJECT_NAME=$4
   DATASET_NAME=$5
-  SOURCE_TABLE_NAME=$6
+  SOURCE=$6
```

Comment:
Yes, using a common variable SOURCE to reuse the existing methods. The values passed to this param is different: tableId in case of bounded mode and GCS URI in case of unbounded mode.

Reply:
Lets add this as a comment.